assets/js/client-mocks.js

```diff
@@ -0,0 +1,1907 @@
+export const codeListMock = (scheme) => {
```

Comment:
I didn't think this file (the mocks for the dropdowns) `assets/js/client-mocks.js` was needed anymore. Not sure if you are intentionally adding them back in?

Reply:
@kdid I didn't intentionally add it (I don't think), but if it's not needed then yea I'll get rid of it.